### PR TITLE
TACO-458 - Fixed slot targeting

### PR DESCRIPTION
**Pull request by domjanik**
Currently, we’ve a problem with retrieving slot targeting. When a user enters a page on mobile, we completely ignore the targeting set for mobile_top_leaderboard. That’s why some values, like testgroups are never available in our slots. In this PR I’ve added code responsible for merging slot-specific targeting with default slot targeting ( mobile_top_leaderboard + top_leaderboard ) so all values would be available.

### CODE REVIEW

File: src/core/models/ad-slot.ts [hunk 1]
```diff
@@ -309,7 +316,7 @@ export class AdSlot {
 				result[key] = value;
 			}
 		});
-
+		utils.logger('DJ', result);
```
Comment (adi-wesolowski-fandom) [inline src/core/models/ad-slot.ts:319]: ?

File: src/core/models/ad-slot.ts [hunk 2]
```diff
@@ -91,7 +91,9 @@ export class AdSlot {
 	viewed: Promise<void> = null;
 
 	constructor(ad: AdStackPayload) {
+		utils.logger('DJ:ad', ad);
```
Comment (adi-wesolowski-fandom) [inline src/core/models/ad-slot.ts:94]: ?

File: src/core/models/ad-slot.ts [hunk 3]
```diff
@@ -91,7 +91,9 @@ export class AdSlot {
...
 		this.config = context.get(`slots.${ad.id}`) || {};
+		utils.logger('DJ:config', this.config);
```
Comment (adi-wesolowski-fandom) [inline src/core/models/ad-slot.ts:96]: ?

**Review by jbienkowski-fandom — APPROVED** [review]
LGTM

**Pull request merged**